scene/gui/text_edit.cpp

```diff
@@ -5718,13 +5718,19 @@ void TextEdit::_update_completion_candidates() {
 
 	bool inquote = false;
 	int first_quote = -1;
+	char restore_quotes = -1;
```

Comment:
Is there any advantage in using a signed char here instead of a plain int?

Reply:
Not really. It can take less space, depending on compiler. But that's not a big advantage by any means so i can change it to something else, no problem.

Reply:
Given that char signedness can be toggled on/off by a compiler flag and is implementation specific, I think using `int` would be safer yes.